.github/scripts/extract_git_ref.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+
+import os
```

Comment:
You only need this if you accept my change below that uses regexes to parse the branch info.

```suggestion
import os
import re
```